### PR TITLE
ER-1197: Swap “description from publisher” and “description from DBC”

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/ER-1197

### CODE REVIEW

File: sites/all/themes/orwell/assets/sass/component/_ting-object.scss [hunk 1]
```diff
@@ -135,7 +135,7 @@ $material--teaser__margin-bottom: 1.2rem;
 
   &__abstract {
     @include text-paragraph;
-    padding-bottom: 40px;
+    // padding-bottom: 40px;
```
Comment (cableman) [inline sites/all/themes/orwell/assets/sass/component/_ting-object.scss:138]: Comment out code?

File: sites/all/themes/orwell/assets/sass/component/_ting-object.scss [hunk 2]
```diff
@@ -336,13 +336,15 @@ $material--teaser__margin-bottom: 1.2rem;
...
     .field-group-format-wrapper,
     .ting-relations__content {
-      padding: 2rem 0;
-
-      @include grid-media($medium) {
-        padding: 0;
-        float: right;
-        display: block !important;
-        width: percentage(2 / 3);
+      // padding: 2rem 0;
```
Comment (cableman) [inline sites/all/themes/orwell/assets/sass/component/_ting-object.scss:339]: Comment out code?

File: sites/all/themes/orwell/assets/sass/component/_ting-object.scss [hunk 3]
```diff
@@ -364,7 +366,7 @@ $material--teaser__margin-bottom: 1.2rem;
     width: 100%;
     display: inline-block;
     clear: left;
-    padding: 1.5rem 0;
+    // padding: 1.5rem 0;
```
Comment (cableman) [inline sites/all/themes/orwell/assets/sass/component/_ting-object.scss:369]: Comment out code?

**Review by cableman — CHANGES_REQUESTED** [review]

File: sites/all/themes/orwell/assets/sass/component/_ting-object.scss (hunk 3 above)
Comment (rimi-itk) [reply sites/all/themes/orwell/assets/sass/component/_ting-object.scss]: Fixed.

**Pull request merged**